src/main/java/uk/gov/hmcts/reform/unspec/validation/RequestExtensionValidator.java

```diff
@@ -0,0 +1,57 @@
+package uk.gov.hmcts.reform.unspec.validation;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.collect.ImmutableList;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.time.LocalDate.now;
+import static java.util.Collections.emptyList;
+
+@Service
+@RequiredArgsConstructor
+public class RequestExtensionValidator {
+
+    private final ObjectMapper mapper;
+
+    public List<String> validateProposedDeadline(CaseDetails caseDetails) {
+        List<String> errors = new ArrayList<>();
+        LocalDate proposedDeadline = mapper.convertValue(
+            caseDetails.getData().get("extensionProposedDeadline"),
+            LocalDate.class
+        );
+
+        if (!proposedDeadline.isAfter(now())) {
+            errors.add("The proposed deadline must be a future date.");
```

Comment:
is the content here confirmed? I was adding CONTENT TBC to make it really obvious where things weren't confirmed :)

Reply:
added TBC to contents to be finalised.